app/controllers/gig_payments_controller.rb

```diff
@@ -0,0 +1,12 @@
+class GigPaymentsController < ApplicationController
+
+  def set_complete
```

Comment:
It wasn't requested by the practice and if I would do it it was a generic PUT EP and not for specific use case like this